Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -989,18 +1131,111 @@ protected int GetVoiceCommandIndex(string command)
 
         #endregion VoiceCommands
 
+        #region NearInteractionHandlers
+        /// <summary>
+        /// Is the eventData a grip action or near pointer?
+        /// </summary>
+        /// <param name="eventData"></param>
+        /// <returns></returns>
+        protected bool IsGrip(IMixedRealityPointer pointer, MixedRealityInputAction action)
+        {
+            if (action.Description.ToLower().IndexOf("grip") > -1 || pointer as IMixedRealityNearPointer != null)
+            {
+                return true;
+            }
+
+            return false;
+        }
+
         void IMixedRealityTouchHandler.OnTouchStarted(HandTrackingInputEventData eventData)
         {
             SetPress(true);
+            if (PhysicalPressOnClickTrigger != PhysicalPressClickBehavior.None)
+            {
+                SetPhysicalTouch(true);
+            }
+
             eventData.Use();
+
+            pressStartPosition = eventData.InputData;
```

Comment:
Again, this sort of pointer management and state management should be shared between the pressable button, so that we don't have duplicate state management code in our MRTK (for when we need to fix bugs).

Reply:
removed